src/psyclone/tests/psyir/backend/fortran_test.py

```diff
@@ -640,7 +640,7 @@ def test_fw_codeblock():
 
 
 @pytest.mark.xfail(reason="issue #430 : module name should be specified")
-def test_module_name(monkeypatch):
```

Comment:
pylint complains:

    fortran_test.py:42:0: C0414: Import alias does not rename original package (useless-import-alias)

Reply:
I can't get my pylint (1.9.5) to complain, but I've modified the code in any case.

Reply:
You've fixed it. FWIW, my pylint is 2.1.1.